v1.6.2/backing-image-manager/rockcraft.yaml

```diff
@@ -0,0 +1,144 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+# Rockcraft definition for the Longhorn Backing Image Manager image:
+# longhornio/backing-image-manager:v1.6.2
+
+name: backing-image-manager
+summary: Rock containing the Longhorn Backing Image Manager component.
+description: |
+  Rock containing Longhorn Backing Image manager component: https://github.com/longhorn/backing-image-manager
+  Aims to replicate the upstream official image: longhornio/backing-image-manager:v1.6.2
+license: Apache-2.0
+
+version: "v1.6.2"
+
+# NOTE(aznashwan): the base for the image is the Suse Linux Enterprise
+# Base Container Image (SLE BCE) Service Pack 6 which ships with Linux 6.4,
```

Comment:
just FYI, I think v1.6.2 is based on SP5.

Reply:
Great catch; I only just noticed that 15.5 is using Linux Kernel 5.x; updating the rockfiles to switch to 22.04 now.